### PR TITLE
Use Django's PBKDF2+bcrypt for password hashing.

### PR DESCRIPTION
**Pull request by akatsoulas**
* Use Django's bcrypt+sha256.

### CODE REVIEW

**Comment by akatsoulas** [comment]
Fixes #114

File: moderator/settings.py [hunk 1]
```diff
@@ -122,7 +121,7 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 
 HMAC_KEYS = config('HMAC_KEYS', cast=json.loads)
 
-PASSWORD_HASHERS = get_password_hashers(BASE_PASSWORD_HASHERS, HMAC_KEYS)
+PASSWORD_HASHERS = get_password_hashers(PASSWORD_HASHERS, HMAC_KEYS)
```
Comment (johngian) [inline moderator/settings.py]: Do we still need that line?

File: moderator/settings.py [hunk 2]
```diff
@@ -122,7 +121,7 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 
 HMAC_KEYS = config('HMAC_KEYS', cast=json.loads)
```
Comment (johngian) [inline moderator/settings.py]: Please remove this line two, if it's not needed.

File: moderator/settings.py (hunk 1 above)
Comment (akatsoulas) [inline moderator/settings.py]: We need to keep these settings in order to be able to upgrade.
https://docs.djangoproject.com/en/1.8/topics/auth/passwords/#password-upgrading

**Comment by akatsoulas** [comment]
@johngian pr updated

File: moderator/settings.py (hunk 1 above)
Comment (johngian) [inline moderator/settings.py]: I don't think that we store password because of persona, but let's be consistent.

**Comment by johngian** [comment]
Code looks OK.

**Comment by johngian** [comment]
r+

**Pull request merged**